src/Item_Project.php

```diff
@@ -265,16 +269,106 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
 
     public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtemplate = 0)
     {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
 
         switch ($item->getType()) {
             case 'Project':
                 self::showForProject($item);
                 break;
 
             default:
-               // Not defined and used now
-               // Project::showListForItem($item);
+                if (in_array($item->getType(), $CFG_GLPI["project_asset_types"])) {
+                    self::showForAsset($item);
+                }
+                break;
         }
         return true;
     }
+
+    private static function showForAsset(CommonDBTM $item): void
+    {
+
+        $item_project = new self();
+        $item_projects = $item_project->find([
+            'itemtype' => $item->getType(),
+            'items_id' => $item->getID()
+        ]);
+
+        $used = $entries = [];
+
+        foreach ($item_projects as $value) {
+            $used[] = $value['projects_id'];
+            $project = new Project();
+            $result = $project->getFromDB($value['projects_id']);
+
+            if ($result === false) {
+                continue;
+            }
+
+            $priority = CommonITILObject::getPriorityName($project->fields['priority']);
+            $state = ProjectState::getById($project->fields['projectstates_id']);
+
+            $data = [
+                'name' => $project->getLink(),
+                'projectstates_id' => $state !== false
+                        ? sprintf(
+                            '<div class="badge_block" style="border-color:%s"><span class="me-1" style="background:%s"></span>%s',
```

Comment:
You don't need HTML in PHP classes, you need to transfer it to TWIG (perhaps by adding macros or adapting some of them).